packages/core/src/components/Button/Button.tsx

```diff
@@ -263,6 +263,7 @@ const Button: VibeComponent<ButtonProps, unknown> & {
         onBlur,
         tabIndex: disabled || ariaHidden ? -1 : tabIndex,
         "data-testid": dataTestId || getTestId(ComponentDefaultTestId.BUTTON, id),
+        "data-vibe-id": getTestId(ComponentDefaultTestId.BUTTON),
```

Comment:
You don't need getTestId as it combines a provided `id` with the test-id, the goal of this property is to be static so it's not needed
```suggestion
        "data-vibe":ComponentDefaultTestId.BUTTON,
```